sdks/wasm/src/library_mono.js

```diff
@@ -375,7 +375,7 @@ var MonoSupportLib = {
 					heapBytes.set (asm);
 					mono_wasm_add_assembly (file_name, memory, asm.length);
 
-					console.log ("MONO_WASM: Loaded: " + file_name);
+					//console.log ("MONO_WASM: Loaded: " + file_name);
```

Comment:
The Loaded message here can actually be quite helpful so we may need to think of a way to make it configurable.  Does not affect nor stop this PR but wanted to point it out.